homeassistant/const.py

```diff
@@ -165,6 +165,11 @@
 EVENT_LOGBOOK_ENTRY = 'logbook_entry'
 EVENT_THEMES_UPDATED = 'themes_updated'
 
+# #### INTENTS ####
```

Comment:
I think it's fine to keep these defined inside helpers.intent .

Reply:
That works, components can just import them from intents, just felt a little cleaner to put the in consts but all good.